js/envirophat/README.md

```diff
@@ -0,0 +1,69 @@
+# Parts list:
+* Raspberry Pi Zero W  (https://www.adafruit.com/product/3410) $34.5 
+* You will USB powered hub, USB keyboard, USB mouse, HDMI cable and a Monitor with HDMI
+* Pimoroni Enviro Phat sensor and actuator board (https://www.adafruit.com/product/3194) $19.95 
+# Hardware Instructions:
+* Solder the headers
+* Connect the Enviro Phat to the GPIO header on the Raspberry Pi
+* Connect the keyboard, mouse and monitor to the Raspberry Pi and insert the SD card with NOOBS installed on it (you can download noobs from (https://downloads.raspberrypi.org/NOOBS_latest) 
+* Connect the microUSB power supply to the microUSB connecter closest to the corner of the board
+# Software Installation Instructions:
+1. Install Raspbian.
+1. create the workspace directory 
+    1. cd ~
+    1. mkdir workspace
+    1. cd workspace
+1. Connect to the Internet (via Ethernet or Wi-Fi)
+1. Install the development environment and IoTivity using the following instructions: 
+    1. sudo apt-get install scons libtool autoconf valgrind doxygen
+    1. sudo apt-get install libboost-dev libboost-program-options-dev libboost-thread-dev uuid-dev libexpat1-dev libglib2.0-dev libsqlite3-dev libcurl4-gnutls-dev libssl-dev
+    1. download iotivity 
+        1. open a browser and go to www.iotivity.org
+        1. click on the link in “Latest Downloads”
+        1. click on “Source Download” (zip or tar.gz should both work)
+        1. open the zip archive and copy the iotivity folder to ~/workspace/
+    1. cd ~/workspace/iotivity-1.3.0 (or whatever version of iotivity was installed)
+    1. build IoTivity using scons 
+        1. scons TARGET_ARCH=arm WITH_RA=1 WITH_RA_IBB=1
+        1. if you are asked to install cbor, use the line provided in the message and run scons again
+        1. if you are asked to install mbedtls, use the line provided in the message and run scons again
+    1. install nvm 
+        1. sudo apt-get update
+        1. curl https://raw.githubusercontent.com/creationix/nvm/v0.33.2/install.sh | bash (or whatever version is latest)
+        1. close the terminal window and reopen
+    1. Use nodejs version 8.1.3: 
+        1. Cd ~/workspace/iotivity-1.3.0
```

Comment:
You don't need to cd into this directory. It won't be available anyway if you leave out the above step.